yugabyte/src/yugabyte/ysql/append.clj

```diff
@@ -98,13 +92,13 @@
   micro-op."
   [conn test [f k v]]
   (let [table (table-for test k)
-        row   (row-for test k)
-        col   (col-for test k)]
+        row (row-for test k)
+        col (col-for test k)]
     [f k (case f
-           :r       (read-primary     conn table row col)
-           :append  (append-primary!  conn table row col v))]))
+           :r (read-primary conn table row col)
+           :append (append-primary! conn table row col v))]))
```

Comment:
This indentation change seems to make things worse, could you try adjusting an auto-formatter?